src/actors/create_container/create_container.py

```diff
@@ -8,10 +8,14 @@ def _execute(cmd):
     return Popen(shlex.split(cmd), stdout=PIPE, stderr=PIPE).communicate()
 
 
-def _build_cmd(source_path, name, img, init_bin, exposed_ports):
+def _build_cmd(source_path, name, version, force, exposed_ports):
     good_mounts = ['bin', 'etc', 'home', 'lib', 'lib64', 'media',
                    'opt', 'root', 'sbin', 'srv', 'usr', 'var']
 
+
+    if force:
```

Comment:
The name of the method is build_cmd, but you are actually executing command here for removing an already created image. I believe this should be done elsewhere